tests/test-war-smoke/src/main/docker/Dockerfile

```diff
@@ -0,0 +1,2 @@
+FROM ${jetty.test.image}
+ADD root /app
```

Comment:
that would be because there is nothing in the war that maps to /, just the hello servlet on /hello

I'll push something on that soon